plugins/clusters.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: clusters
```

Comment:
Maybe you can take inspiration from SIMD (single instruction multiple data). For example `simc` -- _single instruction multiple contexts_.